resources/report-table-formatter.js

```diff
@@ -0,0 +1,85 @@
+{
+    "pangolin strain (#SNPs)": function format(value) {
+        $(function () {
+            $('[data-toggle="tooltip"]').tooltip()
+        })
+
+        if (value !== "no strain called") {
+            var lineage = value.split(' ')[0];
+            var link = "<a data-toggle=\"tooltip\" data-placement=\"top\" title=\"Linkout to cov-lineages\" href='https://cov-lineages.org/lineages/lineage_" + lineage + ".html' target='_blank'>" + lineage + "</a>";
+            return link + value.split(lineage).pop();
+        } else {
+            return value;3
+        }
+    },
+    "variants of interest": function format(value) {
```

Comment:
This function should be basically used twice, for "variants of interest" and "other variants". However, for "variants of interest", it should not only display the gene but also the alteration, i.e. `S:V600E`. Popover should yield the same table as now already. Just more detail next to the gene name. For "other variants" it should stay as it is. Ideally, do not duplicate the function but rather make this configurable via argument, and delegate to the generic function from the two column specific functions.

Reply:
Implemented in eb7a6f6. In order to work we would need https://github.com/rust-bio/rust-bio-tools/pull/142 to be merged though.